Packages/vcs/Lib/vcsvtk/pipeline1d.py

```diff
@@ -39,6 +110,24 @@ def plot(self, data1, data2, tmpl, grid, transform):
         if self._gm.smooth is not None:
             Y = smooth(Y, self._gm.smooth)
 
+        # Here we try to apply the axis convert options
+        for ax, conv, vals in [['x', self._gm.xaxisconvert, X], ['y', self._gm.yaxisconvert, Y]]:
+            if conv == "log10":
+                new = self.convertAxis(ax, vals, numpy.ma.log10, power10)
+            elif conv == "ln":
+                new = self.convertAxis(ax, vals, numpy.ma.log, numpy.ma.exp)
```

Comment:
Why have a `method` and a `revert_method`?

Can't the `revert_method` be inferred from `method`?